html/index.html

```diff
@@ -3,9 +3,213 @@
+          dignissimos eum
+          illum officia reprehenderit nostrum, alias voluptas deserunt ab beatae nam? Blanditiis voluptatibus non dicta
+          quibusdam.
+        </p>
+        <button>Buy Tickets</button>
+        <a>See All Shows</a>
+      </div>
+    </div>
+    <div>
+      <h2>Services</h2>
+      <div>
+        <h3>Corporate Workshops Team Building</h3>
+        <img src='#' alt='#' />
+        <h4>How It Works</h4>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore tempora mollitia soluta doloribus ipsum
+          quibusdam iure quas quasi vitae consequatur sit nam odio, id minima voluptatem magni velit. Distinctio, ex.
+        </p>
+        <h3>Outcome</h3>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati repellendus, quod, eligendi officiis labore
+          magni culpa enim modi impedit amet rem facilis neque dignissimos similique vitae quam consequuntur tempora
+          voluptatum?
+        </p>
+        <button>Request Quote</button>
+      </div>
+      <div>
+        <h3>Improv Classes</h3>
+        <img src='#' alt='#' />
+        <h4>How It Works</h4>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore tempora mollitia soluta doloribus ipsum
+          quibusdam
+          iure quas quasi vitae consequatur sit nam odio, id minima voluptatem magni velit. Distinctio, ex.
+        </p>
+        <h3>Outcome</h3>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati repellendus, quod, eligendi officiis labore
+          magni
+          culpa enim modi impedit amet rem facilis neque dignissimos similique vitae quam consequuntur tempora
+          voluptatum?
+        </p>
+        <button>Sign Up</button>
+      </div>
+      <div>
+        <h3>TV/Film Production</h3>
+        <img src='#' alt='#' />
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nam perferendis fugit doloribus qui magni ratione
+          nisi assumenda officiis explicabo asperiores dolorem ipsam voluptatibus libero harum consectetur et, inventore
+          commodi cum.
+        </p>
+        <ol>
+          <li>Directing</li>
+          <li>Editing</li>
+          <li>Lighting</li>
+          <li>Production Assistant</li>
+        </ol>
+        <button>Contact Me</button>
+      </div>
+      <div>
+        <h3>Acting</h3>
+        <img src='#' alt='#' />
+        <p>
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Enim itaque adipisci alias sed atque quas dolore,
+          neque quasi nulla eos quaerat quae tempore amet laboriosam delectus aspernatur voluptatibus. Suscipit,
+          placeat!
+        </p>
+        <button>Contact Me</button>
+      </div>
+    </div>
+    <div>
+      <h2>Gallery</h2>
+      <div>
+        <h3>Videos</h3>
+        <img src='#' alt='#' />
```

Comment:
these should be in a `<ul>` also this will depend on the responsive slide plugin you decide to go with in regards to syntax.